docs/content_management/pages/page_block_attributes.md

```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
 
 ``` yaml
 [[= include_file('code_samples/page/custom_page_block/config/packages/nested_attribute.yaml', 9,16) =]] [[= include_file('code_samples/page/custom_page_block/config/packages/nested_attribute.yaml', 19,26) =]]
-```
\ No newline at end of file
+```
+
+## Help messages for form fields
+
+With the `help`, `help_attr` and `help_html` field options, you can define help messages for fields in the Page block.
+
+You can set options with the following configuration:
+
+```yaml
+ibexa_fieldtype_page:
+    blocks:
+        block_name:
+            attributes:
+                attribute_name:
+                    options:
+                        help:
+                            text: 'Some exemplary text'
+                            html: true|false
+                            attr:
+                                class: 'class1 class2'
+```
+
+- `help` - defines a help message which is rendered below the field.
+- `help_attr` - sets the HTML attributes for the element which displays the help message.
+- `help_html` - default value: `false`, if set to `true`, the help text is rendered.
```

Comment:
For `help_html` option I will suggest to use similar description as in the official Symfony documentation e.g.: https://symfony.com/doc/current/reference/forms/types/text.html#help-html